src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -53,9 +84,10 @@ def remote_client_int() -> PFClient:
     return PFClient(ml_client=client)
```

Comment:
Seems we have many other clients `rempte_client_int`, `ml_client_canary` etc. Should we cover all of them?

Reply:
updated, no need for that change any more